### PR TITLE
Amend AWS changes

### PR DESCRIPTION
**Pull request by stsoe**

### CODE REVIEW

**Comment by stsoe** [comment]
This cannot be merge before coordinating with build pipeline to change packaging step.
Need to discuss with Nasser and team

File: src/CMake/cpack.cmake [hunk 1]
```diff
@@ -21,16 +25,34 @@ SET(PACKAGE_KIND "TGZ")
...
+  set(CPACK_RPM_XRT_PACKAGE_NAME "xrt")
+
   SET(CPACK_RPM_POST_INSTALL_SCRIPT_FILE "${CMAKE_CURRENT_BINARY_DIR}/postinst")
   SET(CPACK_RPM_PRE_UNINSTALL_SCRIPT_FILE "${CMAKE_CURRENT_BINARY_DIR}/prerm")
+  SET(CPACK_RPM_AWS_POST_INSTALL_SCRIPT_FILE "${CMAKE_CURRENT_BINARY_DIR}/postinst-aws")
+  SET(CPACK_RPM_AWS_PRE_UNINSTALL_SCRIPT_FILE "${CMAKE_CURRENT_BINARY_DIR}/prerm-aws")
   SET(CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "/usr/local" "/usr/src" "/opt" "/etc/OpenCL" "/etc/OpenCL/vendors" "/usr/lib" "/usr/lib/pkgconfig" "/usr/lib64/pkgconfig")
-  set(CPACK_RPM_PACKAGE_REQUIRES "ocl-icd-devel >= 2.2, boost-devel >= 1.53, boost-filesystem >= 1.53, libuuid-devel >= 2.23.2, dkms >= 2.5.0, protobuf-devel >= 2.5.0, protobuf-compiler >= 2.5.0, ncurses-devel >= 5.9, redhat-lsb-core, libxml2-devel >= 2.9.1, libyaml-devel >= 0.1.4 ")
+  # aws component dependencies
+  SET(CPACK_RPM_AWS_PACKAGE_REQUIRES "xrt >= 2.1.0")
```
Comment (rradjabi) [inline src/CMake/cpack.cmake]: Instead of 2.1.0 could we use 
`"xrt >= ${XRT_VERSION_MAJOR}.${XRT_VERSION_MINOR}.${XRT_VERSION_PATCH}" `
instead?

Comment (stsoe) [reply src/CMake/cpack.cmake]: Yes, good suggestion.  Thanks!

**Comment by stsoe** [comment]
retest this please

**Pull request merged**